recipe/meta.yaml

```
@@ -11,7 +11,7 @@ source:
 build:
   skip: true  # [not linux]
   skip: true  # [cuda_compiler_version not in ("11.8", "12.8")]
-  number: 5
+  number: 6
   run_exports:
     - {{ pin_subpackage("ucc") }}
```

Think addressing Peter's original concern would amount to ignoring the `run_export` from `ucx`, and using our own pin, which seems fine for `ucc` to do given it uses the non-public API

```suggestion
  ignore_run_exports_from:
    - ucx
  run_exports:
    - {{ pin_subpackage("ucc") }}
```